src/aiida/__init__.py

```diff
@@ -27,7 +27,7 @@
     'For further information please visit http://www.aiida.net/. All rights reserved.'
 )
 __license__ = 'MIT license, see LICENSE.txt file.'
-__version__ = '2.6.3.post0'
+__version__ = '2.7.0pre1'
```

Comment:
Should this be?
```suggestion
__version__ = '2.7.0rc1'
```

Reply:
So pypi does not anymore support `pre` officially, when released it is automatically converted to `rc0` last time but I started this usage `preX` so I thought for these preleases I just stay consistent and use `preX`. No strong feelings about either way.

Reply:
I think it is very confusing to have a version of pypi different `__version__`.

Reply:
Yes but it is also confusing to have a history of 
```
2.7.0pre0
2.7.0rc1
```
so what do you prefer? Next time I can start with rc0

Reply:
Well hopefully not many people are installing these versions so ultimately it doesn't matter I guess. :-) no strong opinions, with preference to switch to rc1 now.